### ISSUE TITLE
Can't locate Test2/Bundle/More.pm

### ISSUE THREAD

**Issue by eserte** [pherkin/test-bdd-cucumber-perl#158]
It seems that a dependency is not declared in 0.660002-TRIAL:
```
PERL_DL_NONLAZY=1 "/usr/local/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
Can't locate Test2/Bundle/More.pm in @INC (you may need to install the Test2::Bundle::More module) (@INC contains: ... .) at /home/cpansand/.cpan/build/2019092506/Test-BDD-Cucumber-0.660002-3/blib/lib/Test/BDD/Cucumber/Executor.pm line 36, <DATA> line 3514.
BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2019092506/Test-BDD-Cucumber-0.660002-3/blib/lib/Test/BDD/Cucumber/Executor.pm line 36, <DATA> line 3514.
Compilation failed in require at t/210_background_sections.t line 9, <DATA> line 3514.
BEGIN failed--compilation aborted at t/210_background_sections.t line 9, <DATA> line 3514.
t/210_background_sections.t ....................... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
... (etc) ...
```

**Comment by ehuelsmann** [comment]
Thanks for the heads up! Fixed in 9275d26.

**Comment by ehuelsmann** [comment]
(And uploaded as 0.660003-TRIAL)
